scripting/randomizer/sdkhook.sp

```diff
@@ -252,11 +252,6 @@ public Action Client_WeaponEquip(int iClient, int iWeapon)
 {
 	SetEntPropEnt(iWeapon, Prop_Send, "m_hOwnerEntity", iClient);	//So client's class can be attempted first for TF2_GetDefaultClassFromItem
 	
-	ViewModels_UpdateArms(iClient, iWeapon);	// Set arms for the weapon were about to equip
-	
-	//Change class before equipping the weapon, otherwise anims and reload times are odd
-	SetClientClass(iClient, TF2_GetDefaultClassFromItem(iWeapon));
```

Comment:
This fix was added before the whole viewmodel update. While most anims and reload times does seems to have fixed itself from the update, It's however still needed for the first draw animation (along with `ViewModels_UpdateArms` in this hook). Comment could've been updated to reflect this current situation, or whenever if there is another solution to fix this without needing a class change.

Reply:
I didn't see any changes after removing it. Any weapons in particular that get affected by it?

Reply:
Any weapons, only happens when switching to it for the first time.

https://user-images.githubusercontent.com/33488710/222980898-2e599cbf-83e8-4750-874e-3db1a1b27235.mp4